packages/next-on-pages/templates/_worker.js/utils/cache.ts

```diff
@@ -0,0 +1,79 @@
+import {
+	CacheAdaptor,
+	IncrementalCacheValue,
+	SUSPENSE_CACHE_URL,
+} from '../../cache';
```

Comment:
```suggestion
import type {
	CacheAdaptor,
	IncrementalCacheValue} from '../../cache';
import {
	SUSPENSE_CACHE_URL,
} from '../../cache';
```

(PS: these are casing my IDE's eslint to complain, I think it's valid, if I remember correctly we aren't including the templates dir in the eslint config, I need to fix that)